ui/widgets/selectmenu.js

```diff
@@ -354,7 +354,12 @@ return $.widget( "ui.selectmenu", [ $.ui.formResetMixin, {
 		if ( item.disabled ) {
 			this._addClass( li, null, "ui-state-disabled" );
 		}
-		this._setText( wrapper, item.label );
+
+		if ( item.hidden ) {
+			li.prop( "hidden", true );
```

Comment:
I'm not sure about adding hidden prop here. We don't do that anywhere else. But it seems there is no other usage of the hidden attribute, so there is nothing to compare it to directly. 

What about using "aria-hidden" attribute and "ui-helper-hidden" CSS class?

Reply:
@fnagel Sure, I can change the code. The behavior in IE 9 & 10 will be different but perhaps if the `hidden` attribute is there, it's expected that we should hide it even if the browser doesn't support it natively.

One thing - I see the `ui-helper-hidden` class definition in CSS but other than that, I don't see it anywhere in code; only the `ui-helper-hidden-accessible` one seems to be used. Am I missing anything?

Reply:
I'm not sure about my proposed change either :-D Just wanted to point out our options and get your opinion on it.

You are right, `ui-helper-hidden` is not used in JS. And `ui-helper-hidden-accessible` seems not correct here. 

Maybe its best to use it as is it is (with the hidden property) and let the client (or its assistive technology) decide what to do.

Reply:
In that case, I think I'd leave it at `hidden`. @fnagel do you have other concerns? If not, could you approve?